tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -118,10 +118,14 @@ namespace tensorflow {
 //      take (A, B, A_m, B_m) as input. Among N inputs one can get N!
```

Comment:
This file is getting very long. Maybe separate all declarations to mkl_layout_pass.h, that would make it easier to read.